conda_forge_tick/update_upstream_versions.py

```diff
@@ -114,6 +114,26 @@ def get_version(self, url):
         return r.json()["info"]["version"].strip()
 
 
+class NPM:
+    name = "npm"
+
+    def get_url(self, meta_yaml):
+        if "registry.npmjs.org" in meta_yaml["url"]:
+            return None
+        # might be namespaced
+        pkg = meta_yaml["url"].split("/")[3:-2]
+        return "https://registry.npmjs.org/{}".format("/".join(pkg))
+
+    def get_version(self, url):
+        r = requests.get(url)
+        latest = r.json()["dist-tags"].get("latest", "").strip()
```

Comment:
Can you please check `r.ok` before trying to get the json?

Reply:
fixed!